test/common/io/io_uring_impl_test.cc

```diff
@@ -0,0 +1,239 @@
+#include <sys/epoll.h>
```

Comment:
Please look at the test failures in CI:

[2021-12-28 13:52:10.547][15][critical][assert] [source/common/io/io_uring_impl.cc:34] assert failure: ret == 0. Details: unable to initialize io_uring: Operation not permitted

Reply:
This is complicated.

The test passes perfectly when run with Azure's managed workers: https://dev.azure.com/dmitryrojkov/meta-helloworld/_build/results?buildId=13&view=logs&j=8c169225-0ae8-53bd-947f-07cb81846cb5&t=6b0ace90-28b2-51d9-2951-b1fd35e67dec
It fails if RBE is active. I suspect that io_uring syscalls are not allow-listed on the workers used as RBEs. Do you happen to know how to reach out to people running those builders?

Reply:
cc @yanavlasov who may have some thoughts.